network-mapper/README.md

```diff
@@ -1,13 +1,17 @@
 # Parameters
 
 ## Mapper parameters
-| Key                       | Description              | Default          |
-|---------------------------|--------------------------|------------------|
-| `mapper.image.repository` | Mapper image repository. | `otterize`       |
-| `mapper.image.image`      | Mapper image.            | `network-mapper` |
-| `mapper.image.tag`        | Mapper image tag.        | `latest`         |
-| `mapper.pullPolicy`       | Mapper pull policy.      | `(none)`         |
-| `mapper.resources`        | Resources override.      | `(none)`         |   
+| Key                            | Description                          | Default                        |
+|--------------------------------|--------------------------------------|--------------------------------|
+| `mapper.image.repository`      | Mapper image repository.             | `otterize`                     |
+| `mapper.image.image`           | Mapper image.                        | `network-mapper`               |
+| `mapper.image.tag`             | Mapper image tag.                    | `latest`                       |
+| `mapper.pullPolicy`            | Mapper pull policy.                  | `(none)`                       |
+| `mapper.resources`             | Resources override.                  | `(none)`                       |
+| `mapper.apiAddress`            | Api address.                         | `https://app.otterize.com/api` |
+| `mapper.clientSecret`          | Cloud client secret.                 | `(none)`                       |
+| `mapper.clientid`              | Cloud client id.                     | `(none)`                       |
```

Comment:
These settings should be for the entire chart, not the mapper - it's an implementation detail that it's just the mapper.
Also, the docs should say something like - Client ID for connecting to Otterize Cloud, Client Secret for connecting to Otterize.